Source/BSN.IpTables.Cli/custom/Module.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace BSN.IpTables.V1
+{
+    public partial class Module
+    {
+        partial void AfterCreatePipeline(global::System.Management.Automation.InvocationInfo invocationInfo,
+            ref BSN.IpTables.V1.Runtime.HttpPipeline pipeline)
+        {
+            if (pipeline == null)
+                throw new NullReferenceException("Pipeline is null!");
+            pipeline.Append(SendAsync);
+        }
+
+        partial void BeforeCreatePipeline(global::System.Management.Automation.InvocationInfo invocationInfo,
+            ref BSN.IpTables.V1.Runtime.HttpPipeline pipeline)
+        {
+        }
+
+        public async System.Threading.Tasks.Task<System.Net.Http.HttpResponseMessage>
+           SendAsync(System.Net.Http.HttpRequestMessage request,
+            BSN.IpTables.V1.Runtime.IEventListener callback,
+            BSN.IpTables.V1.Runtime.ISendAsync next)
+        {
+            string serverAddress = Environment.GetEnvironmentVariable("ServerAddress").ToString();
```

Comment:
I think we have to check `ServerAddress` exist or not?

and corresponding log message (or error message)

Reply:
I do not sure `Console.WriteLine` is best choice for logging

@siadatism

Reply:
In this case, I suggest throwing exception with informative message.

Reply:
Done.